includes/class-content-distribution.php

```diff
@@ -109,6 +110,42 @@ public static function get_distributed_post_types() {
 		return apply_filters( 'newspack_network_distributed_post_types', [ 'post' ] );
 	}
 
+	/**
+	 * Get post meta keys that should be ignored on content distribution.
+	 *
+	 * @return string[] The reserved post meta keys.
+	 */
+	public static function get_reserved_post_meta_keys() {
+		$reserved_keys = [
+			'_edit_lock',
+			'_edit_last',
+			'_thumbnail_id',
+			'_yoast_wpseo_primary_category',
+		];
+
+		/**
+		 * Filters the reserved post meta keys that should not be distributed.
+		 *
+		 * @param string[] $reserved_keys The reserved post meta keys.
+		 * @param WP_Post  $post          The post object.
+		 */
+		$reserved_keys = apply_filters( 'newspack_network_content_distribution_reserved_post_meta_keys', $reserved_keys );
+
+		// Always preserve content distribution post meta.
+		$reserved_keys = array_merge(
```

Comment:
Nit: we could also just return the array_merge here. No need for the `$reserved_key` var.

Reply:
Good call! Updated in 53de4f37d70e2beb1414cfb574d1bf2f3bf9000c